build.gradle

```diff
@@ -47,7 +47,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '11'
+    String javaFxVersion = '12'
```

Comment:
Actually may I know the reason for changing it to 12? Am afraid that it might introduce some errors due to the [specifications of this module](https://nus-cs2103-ay2122s1.github.io/website/admin/tp-constraints.html#constraint-java-version)...

Reply:
Nvm saw that it's reverted back in 72d3893!